apps/web/__tests__/support/pageObjects/EditorObject.ts

```diff
@@ -37,6 +37,22 @@ export class EditorObject extends PageObject {
     return cy.get('#close')
   }
 
+  get blockWrapper() {
+    return cy.getByTestId('block-wrapper')
+  }
+
+  get topBlockButton(){
+    return cy.getByTestId('top-add-block')
+  }
+
+  get bottomBlockButton(){
+    return cy.getByTestId('btm-add-block')
```

Comment:
```suggestion
    return cy.getByTestId('bottom-add-block')
```
Please avoid using abbreviations.